.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.x'
```

Comment:
What's GitHub's policy on maintaining these containers (I assume that's what they are)? Because you might want to lock it to Ruby 2.6 if you can (2.7 has some minor breaking changes and is coming out on Christmas, 3.0 will come out next year and have actual breaking changes), and lock the version of the travis gem that you're installing with `gem install travis:1.8.11 --no-document`.

Though I'm also not sure how often the Travis API changes and whether that gem would need to be updated at some point in the future.